app/experimenter/experiments/apps.py

```diff
@@ -8,4 +8,29 @@ class ExperimentsConfig(AppConfig):
     name = "experimenter.experiments"
 
     def ready(self):
+
+        # create user groups and add permissions to them
+        from django.contrib.auth.models import Group, Permission
+        from django.contrib.contenttypes.models import ContentType
+        from experimenter.experiments.models import Experiment
+
         markus.configure(settings.MARKUS_BACKEND)
+
+        relman_signoff_group, created = Group.objects.get_or_create(
```

Comment:
This `created` gets ignored. Is that intentional? I think it's safe-ish to assume that both will be created or neither will be, but maybe we should make this explicit by marking one of these `created` with `_`.

Reply:
@glasserc I actually changed from doing this inside `ready` to doing it in a data migration so this code is gone!